requirements.frozen.txt

```diff
@@ -1,18 +1,19 @@
+asgiref==3.6.0
 autopep8==2.0.1
 certifi==2022.12.7
 charset-normalizer==2.0.12
-Django==2.2.28
+Django==4.1
```

Comment:
We also need to bump Django `requirements.txt`. The way I've been upgrading pip dependencies until now has been to bump `requirements.txt`, creating a fresh venv, installing `requirements.txt` and then do `pip freeze > requirements.frozen.txt`, but I've been meaning to improve on that. Created https://github.com/knatten/cppquiz/issues/281 for it now.

Whether we use the old method in this PR or solve #281 first is both fine with me.

Reply:
https://github.com/knatten/cppquiz/pull/279/commits/d05a4d379b2ab519c157099ba3a52c6eaf1ea08a